Makefile

```diff
@@ -4,54 +4,29 @@
-#   - build-fabric                 - builds fabric docker images and binaries.
-#   - build-fabric-ca              - builds fabric-ca docker images and binaries.
-#   - fabric                       - clones fabric repository.
-#   - fabric-chaincode-java        - clones the fabric-chaincode-java repository.
-#   - smoke-tests                  - runs Smoke Test Suite.
-#   - barebones-tests              - runs Barebones Test Suite.
-#   - daily-tests                  - runs Daily Test Suite.
-#   - pull-images                  - pull the images and binaries from Artifactory.
-#   - javaenv                      - clone the fabric-chaincode-java repository and build the javaenv image.
-#   - nodeenv                      - clone the fabric-chaincode-node repository and build the nodeenv image.
-#   - svt-weekly-pte-12hr-test-k8s - Test 12hr longrun test in k8s environment.
-#   - npm-init                     - initialize the PTE NPM modules
-#   - git-latest                   - init git submodules to latest available commit.
-#   - git-init                     - init git submodules.
-#   - pre-setup                    - installs node and govendor
-#   - clean                        - cleans the docker containers.
-#   - gotools                      - installs go tools, such as: ginkgo, golint, goimports, gocov and govendor
+#   - regression/barebones			- Executes barebones tests
+#   - regression/basicnetwork		- Executes basicnetwork tests
+#   - regression/smoke				- Executes smoke tests
+#   - regression/systemtest			- Executes system tests on k8s cluster
+#   - regression/weekly				- Executres 12hr long running test in k8s environment
+#   - npm-init						- Initializes the PTE NPM modules
+#   - docker-clean					- Kills and removes all running Docker containers
+#   - gotools						- Installs go tools, such as: ginkgo, golint, goimports, gocov and govendor
 #
 # ------------------------------------------------------------------
 
-BRANCH = master
-FABRIC = https://github.com/hyperledger/fabric
-FABRIC_CA = https://github.com/hyperledger/fabric-ca
-FABRIC-CHAINCODE-JAVA = https://github.com/hyperledger/fabric-chaincode-java
-FABRIC-CHAINCODE-NODE = https://github.com/hyperledger/fabric-chaincode-node
-HYPERLEDGER_DIR = $(GOPATH)/src/github.com/hyperledger
-FABRIC_DIR = $(HYPERLEDGER_DIR)/fabric
-CA_DIR = $(HYPERLEDGER_DIR)/fabric-ca
-CHAINCODE-JAVA_DIR = $(HYPERLEDGER_DIR)/fabric-chaincode-java
-CHAINCODE-NODE_DIR = $(HYPERLEDGER_DIR)/fabric-chaincode-node
-PRE_SETUP = $(GOPATH)/src/github.com/hyperledger/fabric-test/scripts/pre_setup.sh
-
 include gotools.mk
 
-.PHONY: ci-smoke
-ci-smoke: pre-reqs pull-binaries-fabric smoke-tests
+regression/%: pre-reqs
+	cd ${@} && ginkgo -v
 
-.PHONY: ci-barebones
-ci-barebones: pre-reqs pull-binaries-fabric barebones-tests
+.PHONY: pre-reqs
+pre-reqs: docker-clean npm-init pull-binaries-fabric gotools
 
-.PHONY: ci-daily
-ci-daily: pre-reqs pull-binaries-fabric daily-tests
+.PHONY: docker-clean
```

Comment:
Where are we using docker-clean target? In test suites, we take care of fabric networks and chaincode containers in AfterSuite.

Reply:
Removed, I don't like it anyway, it might clean up containers I am running on my own